scripts/deployment/deploy-basic-sell.ts

```diff
@@ -0,0 +1,31 @@
+import hre, { ethers } from 'hardhat'
+import { HardhatUtils, TriggerType, deployCommand, ensureEntryInServiceRegistry } from '../common'
+
+async function main() {
```

Comment:
i see that MCDVat is missing from this script, but was added to deploy system. in general, let's merge the deployment scripts for both basic buy and basic sell